02_calculator/calculator_spec.rb

```diff
@@ -79,21 +79,37 @@
 # once the above tests pass,
 # write tests and code for the following:
 
-describe '#multiply' do
-  it 'multiplies two numbers'
+describe 'multiply' do
+  it 'multiplies two numbers' do
+	 	 multiply([4, 2]).should == 8
+			end
 
-  it 'multiplies several numbers'
+  it 'multiplies several numbers' do
+		  multiply([5, 1, 2, 3]).should == 30
+		end
 end
 
-describe '#power' do
-  it 'raises one number to the power of another number'
+describe 'power' do
+  it 'raises one number to the power of another number' do
+			power(5, 2).should == 25
+  end
 end
 
 # http://en.wikipedia.org/wiki/Factorial
-describe '#factorial' do
-  it 'computes the factorial of 0'
-  it 'computes the factorial of 1'
-  it 'computes the factorial of 2'
-  it 'computes the factorial of 5'
-  it 'computes the factorial of 10'
+describe 'factorial' do
+  it 'computes the factorial of 0'do
+			factorial(0).should == 1
```

Comment:
Using tabs or using spaces? Doesn't rubocop complain?

Reply:
Yes it does....But because those are not written by me, I focused more on solution files, not on specs